pandas/tslib.pyx

```diff
@@ -2416,6 +2416,8 @@ cdef inline parse_timedelta_string(object ts, coerce=False):
     elif have_dot:
 
         if (len(number) or len(frac)) and not len(unit) and current_unit is None:
+            if coerce:
+                return iNaT
```

Comment:
`coerce` should `coerce` - this was raising when trying to convert `3.1415` when `coerce=True` and was incorrectly raising a ValueError.

The recent commit to tslib broke tests in this PR.